bootstrap/interfaces/configuration.go

```diff
@@ -46,6 +47,6 @@ type Configuration interface {
 	// GetLogLevel returns the current ConfigurationStruct's log level.
 	GetLogLevel() string
 
-	// SetRegistryInfo updates the config.RegistryInfo field in the ConfigurationStruct.
-	SetRegistryInfo(registryInfo config.RegistryInfo)
+	// GetRegistryInfo gets the config.RegistryInfo field from the ConfigurationStruct.
+	GetRegistryInfo() config.RegistryInfo
```

Comment:
This is uncomfortable to me; not sure what follow-on code usage looks like.  Is it no longer necessary to SetRegistryInfo(); how did this morph into GetRegistryInfo()?

Reply:
This is because Registry now only come from the configuration. We are no longer overriding it with the `edgex_registry` environment variable. Bootstrap needs to `Get` RegistryInfo from the configuration in order to connect and register the service.